setup.py

```diff
@@ -292,7 +292,7 @@
 
 setup_kwargs = {
     "name": "pymedphys",
-    "version": "0.38.0.dev4",
+    "version": "0.38.0",
```

Comment:
is this updated automatically from pyproject.toml?

Reply:
Yup, just need to change `pyproject.toml` and it should be propagated through.